packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -447,16 +447,17 @@ export default class ReconnectionManager {
   }
 
   /**
+   * @param {Object} options passes mediaSetting and streams if needed
    * @returns {Promise}
    * @private
    * @memberof ReconnectionManager
    */
-  reconnectMedia() {
+  reconnectMedia(options = {}) {
```

Comment:
if we're adding a new argument, it needs to be documented, but I hope we can remove this - see my other comment in this function